index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Gesture Help Desk</title>
+
+    <meta charset="utf-8">
+
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:300italic,300,600,800" rel="stylesheet" type="text/css">
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/stylesheet.css">
+</head>
+
+<body>
+
+  <header>
+    <img class="coverImg" src="images/gesture-intro.jpg">
```

Comment:
Use cover-img instead of coverImg. Snake your classes together using a dash instead of camel casing.